src/profile.js

```diff
@@ -79,6 +86,39 @@ import {withRouter} from 'react-router-dom'
       });
     }
 
+    checkAge = () => {
+      const birthday_yr = this.state.birthday_yr;
+      const birthday_mo = this.state.birthday_mo;
+      const birthday_dt = this.state.birthday_dt;
+
+      var birthday = Date(birthday_yr, birthday_mo, birthday_dt);
+      
+      var today = new Date();
+      var age = today.getFullYear() - birthday_yr
```

Comment:
since you are checking the age here, can you pass age as a parameter to the backend so you don't need to recalculate age?

Reply:
should be fixed now